pkg/identity/identity_test.go

```diff
@@ -117,7 +111,4 @@ func TestUpdateToken(t *testing.T) {
 	if token1 == token2 || token1Expire == token2Expire {
 		t.Error("Token failed to updated")
 	}
-	if *tp.config.Client.CredsToken != token2 {
-		t.Error("Failed to update client token to token2")
-	}
```

Comment:
Why remove those test cases?

Reply:
Hi @zetaj ,
From the given nested structure `*tp.config.Client.CredsToken`, it looks like the `CredsToken` field has been removed from the recent upgraded packages of `github.com/AthenZ/athenz/clients/go/zms`. As a result, it is no longer supported, hence our test cases are failing. Therefore, we removed the `CredsToken` field.